### ISSUE TITLE
when iput fails it keeps on retrying if no retries count given

### ISSUE THREAD

**Issue by gitjopa** [irods/irods#2615]
irods master 9fd22971468dcecd46008c68cf9d080a6c44f98 (Sat Apr 11 16:08:31 2015 -0400)

OS version:
```
[irods@localhost ~]$ cat /etc/redhat-release 
CentOS release 6.6 (Final)
```

When iput fails it keeps on retrying:
```
[irods@localhost ~]$ iput foo /tempZone/home/rods/does_not_exist/bar
ERROR: putUtil: put error for /tempZone/home/rods/does_not_exist/bar, status = -814000 status = -814000 CAT_UNKNOWN_COLLECTION
ERROR: putUtil: put error for /tempZone/home/rods/does_not_exist/bar, status = -814000 status = -814000 CAT_UNKNOWN_COLLECTION
ERROR: putUtil: retry putUtil error status = -814000 CAT_UNKNOWN_COLLECTION
ERROR: putUtil: put error for /tempZone/home/rods/does_not_exist/bar, status = -814000 status = -814000 CAT_UNKNOWN_COLLECTION
ERROR: putUtil: retry putUtil error status = -814000 CAT_UNKNOWN_COLLECTION
ERROR: putUtil: put error for /tempZone/home/rods/does_not_exist/bar, status = -814000 status = -814000 CAT_UNKNOWN_COLLECTION
ERROR: putUtil: retry putUtil error status = -814000 CAT_UNKNOWN_COLLECTION
^C
```

iput with `--retries` stops as expected:
```
[irods@localhost ~]$ iput --retries 0 foo /tempZone/home/rods/does_not_exist/bar
ERROR: putUtil: put error for /tempZone/home/rods/does_not_exist/bar, status = -814000 status = -814000 CAT_UNKNOWN_COLLECTION
Level 0: collection '/tempZone/home/rods/does_not_exist' is unknown
[irods@localhost ~]$
```

**Comment by kellerb** [comment]
I suspect that was fixed by 2a9293200805b983b56b473ec3fff62579611a3d

**Comment by gitjopa** [comment]
Thanks, it stops on error now.
